package.json

```diff
@@ -20,7 +20,7 @@
     "start-multi2": "cross-env NODE_APP_INSTANCE=2 electron .",
     "start-prod": "cross-env NODE_ENV=production NODE_APP_INSTANCE=devprod electron .",
     "start-prod-multi": "cross-env NODE_ENV=production NODE_APP_INSTANCE=devprod1 electron .",
-    "start-prod-multi-2": "cross-env NODE_ENV=production NODE_APP_INSTANCE=devprod2 electron .",
+    "start-prod-multi1": "cross-env NODE_ENV=production NODE_APP_INSTANCE=devprod2 electron .",
```

Comment:
made a change in my waiting to PR to remove those copy pasted scripts. 
once my PR is merged you can start any number of instances
by running `MULTI=1 start-prod`

https://github.com/loki-project/session-desktop/pull/1137/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R19